test/HeadersTest.php

```diff
@@ -309,6 +309,31 @@ public function testCastingToStringReturnsAllMultiHeaderValues()
         $this->assertEquals($expected, $string);
     }
 
+    /**
+     * @test that toArray can take format parameter
+     * @link https://github.com/zendframework/zend-mail/pull/61
+     */
+    public function testToArrayFormat()
```

Comment:
I was waiting when communication finishes, if the communication ends up whole PR being rejected, i don't see point fixing the tests. so please respond after my replies which changes should be done that you initially mentioned (besides test splitting)